core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/SparkRunner.scala

```diff
@@ -240,11 +378,16 @@ object SparkResource {
       exposeExecutorMetrics: Boolean = true
   )
 
+  /**
+   * NOTE: coreRequest in Driver is only supported in Spark 3.0:
+   * https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/8c480acfdd09882ed2f00573f15e7830558de524/pkg/apis/sparkoperator.k8s.io/v1beta2/types.go#L499
+   */
   final case class Driver(
       cores: Option[Int] = Some(1),
       coreLimit: Option[String] = None,
       memory: Option[String] = None,
       memoryOverhead: Option[String] = None,
+      env: Option[List[EnvVar]] = None,
```

Comment:
How about renaming it to `envs` because it's a list of envs?

Reply:
The name in json has to be env

Reply:
That will not work, mapping to spark https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/8c480acfdd09882ed2f00573f15e7830558de524/pkg/apis/sparkoperator.k8s.io/v1beta2/types.go#L431